### ISSUE TITLE
User name field value not being updated

### ISSUE THREAD

**Issue by snoozan** [jenkinsci/oic-auth-plugin#7]
When attempting to update the `User name field` in the Jenkins UI from `sub` to `preferred_username`, the value is not saved and remains at `sub`. 
![screen shot 2017-02-01 at 2 09 36 pm](https://cloud.githubusercontent.com/assets/3599795/22522070/358f4164-e888-11e6-8532-18e8eac8957c.png)

I tracked this down to `doCheckUserNameField` method:
```
388         public FormValidation doCheckUserNameField(@QueryParameter String userNameField) {
389             if (userNameField == null || userNameField.trim().length() == 0) {
390                 return FormValidation.ok("Using 'sub'.");
391             }
392             return FormValidation.ok();
393         }
```
For some reason, userNameField is null rather than populated with the updated value.

If the `config.xml` is updated with 
```
    <userNameField>sub</userNameField>
    <fullNameFieldName>name</fullNameFieldName>
    <scopes>openid email</scopes>
  </securityRealm>
``` 
 to
```
    <userNameField>preferred_username</userNameField>
    <fullNameFieldName>name</fullNameFieldName>
    <scopes>openid email</scopes>
  </securityRealm>
```
it works fine. However, default values are obviously returned if you attempt to use the Jenkins UI to update the configuration.

**Comment by mjmbischoff** [comment]
Hello Susan, thanks for reporting this.

I think this might have gotten fixed already with https://github.com/jenkinsci/oic-auth-plugin/pull/2 I'll make sure I'll give it a test before running the release. Leaving it open for now.

I hope to finish (at least part) of an other issue and then I'll test and bake a release.

**Comment by aligin** [comment]
Hello. So when the release will be? We expect the username from userInfo functionality and resolving this issue (although it is possible to correct it by hand in the .xml).

**Comment by cgroschupp** [comment]
@mjmbischoff Can you release a new version to fix this issue?

**Comment by cgroschupp** [comment]
@agentgonzo For me it worked.

**Comment by agentgonzo** [comment]
@mjmbischoff @sboschman - is anyone still maintaining this repo? I've recently made some improvements in a fork but have noticed that there has been no active changes on this repo in a year? I'm happy to take over maintaining this repo (and doing a new release with the fixed bugs!) if required.

**Comment by sboschman** [comment]
I am not a maintainer of this repo/plugin, only supplied a patch. So, unfortunately I can't make a release or anything. Made my own build of this plugin which is still being used daily to login to our Jenkins instance. If you are willing to maintain this plugin, you have my blessings.

**Comment by mjmbischoff** [comment]
Fixed in 1.1, 

Anyone who want to become an maintainer is welcome to do so.
